packages/components/src/tooltip/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BaseTooltip from '../base_tooltip';
+import styles from './index.module.scss';
+
+const Tooltip = (props) => {
+  const {
+    position,
+    bubble,
+    children,
+    tooltipClassName,
+    arrowClassName,
+    triggerProps,
```

Comment:
lot of unused stuff here

Reply:
Just can't find this piece of code - super weird

Reply:
I just pulled - should be here https://github.com/goodhood-eu/goodhood/blob/df317c1f55e90108b4f24719630111b4c0127a5d/packages/components/src/tooltip/index.jsx#L11